test-requirements.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/cloudify-cosmo/cloudify-cli@master#egg=cloudify==master
 git+https://github.com/cloudify-cosmo/cloudify-agent@master#egg=cloudify-agent==master
 mock
 passlib==1.7.1
+git+https://github.com/cloudify-cosmo/cloudify-fabric-plugin@master#egg=cloudify-fabric-plugin==master
```

Comment:
plz pin this and the cli to a specific tag

Reply:
CLI probably shouldn't be pinned, as it (at the moment) needs to be master I think?

Reply:
Fabric pinned to 1.6.0.